projects/core/src/auth/http-interceptors/user-token.interceptor.ts

```diff
@@ -32,7 +31,7 @@ export class UserTokenInterceptor implements HttpInterceptor {
   ): Observable<HttpEvent<any>> {
     if (
       this.userToken &&
```

Comment:
hard to read (but not your concern ;) ), but shall we move it to a private method that tells us from signature what this is about?

Reply:
Ok